omod/src/main/web/dashboardwidgets/dashboardwidgetscommons.service.js

```diff
@@ -127,4 +129,33 @@ export default class WidgetsCommons {
         return Boolean(obj.concept);
     }
 
+    /**
+     * Returns date translated according to preferred locale when Date Format displays the month as an abbreviation (MMM).
+     * Ex: Date: 02-Jan-2021 Format: DD-MMM-YYYY Locale: fr --> Return: 02-janv.-2021
+     *
+     * @param {object} date Input Date
+     * @param {string} format Date Format
+     * @param {string} locale The preferred locale
+     */
+    formatDate(date, format, locale) {
+        try{
+            moment.locale(locale);
+            return moment(date).format(format);
+        } catch(err) {
+            return moment(date).format("DD.MMM.YYYY");
+        }
+    }
+
+    /**
+     * Return true if the user has the System Developer role
+     *
+     * @param user
+     * @returns {*}
+     */
+    isSystemDeveloper(user){
+        if(user.roles != null && user.roles != undefined){
+            return user.roles.some( (p) => { return p.name === 'System Developer'; });
+        }
+        return false;
+    }
```

Comment:
@JFFigueiredo why is this null check not on master?

Reply:
I needed to add that check, so the build pass all the tests.. It was not on master, so i only add it here.

Reply:
Maybe with github actions something changed, because, if that code is on master without the null check is because in the past it should be ok...